docs/eventing/sources/sinkbinding.md

```diff
@@ -104,9 +104,11 @@ Create a `CronJob` object:
               restartPolicy: Never
               containers:
                 - name: single-heartbeat
-                  image: <FILL IN YOUR IMAGE HERE>
+                  image: busybox # Or your preferred image
                   args:
-                    - --period=1
+                  - /bin/sh
```

Comment:
I don't think this is a good change, as the example wants to give the user a cronjob example with an image that send a HTTP post cloud event to the to be injected K_SINK which in this case is asking the user to replace with heartbeats image.

Maybe what's need it is in image more info about the heartbeat image location to be inserted, or we can put a predefined image from gcr and skip the step on creating the image.

Reply:
Agreed, to me `busybox` etc doesn't mean anything / isn't helpful here.
Can we work on getting some standard image to use here @csantanapr or can someone provide more info to clarify this? Could you assign an appropriate SME?

Reply:
the heartbeat image is published there: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/heartbeats.

Reply:
I see, I didn't realize the image needed for this chronjob example needed to send an HTTP post cloud event. Busybox doesn't make sense for that.

> Maybe what's need it is in image more info about the heartbeat image location to be inserted

This would help, at least making a specific point that not just any chronjob will do. You need one with the proper cloud event behavior.

Reply:
Like @lionelvillard indicated use the image `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/heartbeats`